test/plots/wheelPlotTests.ts

```diff
@@ -0,0 +1,296 @@
+        assert.isUndefined(wheelPlot.startAngle(), "startAngle is initiized to undefined");
+
+        wheelPlot.startAngle(0);
+        wheelPlot.renderTo(svg);
+        assert.strictEqual(wheelPlot.startAngle().accessor(data[0], 0, dataset), 0, "access startAngle that is set to a constant");
+        assert.strictEqual(wheelPlot.startAngle().accessor(data[1], 1, dataset), 0, "access startAngle that is set to a constant");
+        assert.isUndefined(wheelPlot.startAngle().scale, "scale of startAngle is undefined");
+
+        wheelPlot.startAngle((d) => d.t1);
+        assert.strictEqual(wheelPlot.startAngle().accessor(data[0], 0, dataset), 0, "access startAngle correctly without a scale");
+        assert.strictEqual(wheelPlot.startAngle().accessor(data[1], 1, dataset), 180, "access startAngle correctly without a scale");
+        assert.isUndefined(wheelPlot.startAngle().scale, "scale of startAngle is undefined");
+
+        wheelPlot.startAngle((d) => d.t1, tScale);
+        assert.deepEqual(wheelPlot.startAngle().scale.range(), [0, Math.PI * 2], "range of startAngle should be 0 to 2PI");
+        assert.strictEqual(wheelPlot.startAngle().accessor(data[0], 0, dataset), 0, "access startAngle correctly with a scale");
+        assert.strictEqual(wheelPlot.startAngle().accessor(data[1], 1, dataset), 180, "access startAngle correctly with a scale");
+        assert.deepEqual(wheelPlot.startAngle().scale, tScale, "scale of startAngle is set correctly");
+
+        svg.remove();
+      });
+
+      it("can set and get endAngle", () => {
+        wheelPlot.startAngle((d) => d.t1);
+        assert.isUndefined(wheelPlot.endAngle(), "endAngle is initiized to undefined");
+
+        wheelPlot.endAngle(0);
+        wheelPlot.renderTo(svg);
+        assert.strictEqual(wheelPlot.endAngle().accessor(data[0], 0, dataset), 0, "access endAngle that is set to a constant");
+        assert.strictEqual(wheelPlot.endAngle().accessor(data[1], 1, dataset), 0, "access endAngle that is set to a constant");
+        assert.isUndefined(wheelPlot.endAngle().scale, "scale of endAngle is undefined");
+
+        wheelPlot.endAngle((d) => d.t2);
+        assert.strictEqual(wheelPlot.endAngle().accessor(data[0], 0, dataset), 180, "access endAngle correctly without a scale");
+        assert.strictEqual(wheelPlot.endAngle().accessor(data[1], 1, dataset), 360, "access endAngle correctly without a scale");
+        assert.isUndefined(wheelPlot.endAngle().scale, "scale of endAngle is undefined");
+        svg.remove();
+      });
+
+      it("sets scale of endAngle when scale of startAngle is set", () => {
+        wheelPlot.endAngle((d) => d.t2);
+        wheelPlot.renderTo(svg);
+
+        assert.isUndefined(wheelPlot.endAngle().scale, "scale of endAngle is undefined initially");
+
+        wheelPlot.startAngle((d) => d.t1, tScale);
+        assert.deepEqual(wheelPlot.endAngle().scale, tScale, "scale of endAngle is set to be the same scale of startAngle");
```

Comment:
Might be worth confirming that setting `endAngle()` after setting `startAngle()` still picks up the `Scale`.